src/dll/edbgEcmdDll.C

```diff
@@ -2038,6 +2038,12 @@ uint32_t iStepsHelper(uint16_t i_major_start,
               rc = ipl_run_major_minor(major_istep, minor_istep);
               if (!rc)
               {
+                  if (major_istep >= 6) {
```

Comment:
This will trigger updates in every step after step 6 , we should eb doing only at step 6 begining. or sucesssfull step 5.

Reply:
setHostStateToRunning()  in edbgIstep.C  will internally check if Host is already running before setting state. So, it will set only once and if istep is 6 and above. Do you want to make it 5 instead of 6?